src/views/product-landing/schema.ts

```diff
@@ -44,6 +44,18 @@ const TutorialCardsContentBlockSchema = Joi.object({
   tutorialSlugs: Joi.array().items(Joi.string()).required().min(1),
 })
 
+/**
+ * CollectionCards
+ */
+interface CollectionCardsContentBlock {
+  type: 'collection_cards'
+  collectionSlugs: string[]
+}
+const CollectionCardsContentBlockSchema = Joi.object({
```

Comment:
We add collection cards to the content schema, and to the corresponding content type.